### ISSUE TITLE
Internal Limit(s) reached?

### ISSUE THREAD

**Issue by smmmadden** [KaiKikuchi/QuickShop#118]
I've been converting all my chests to QuickShop and appear to have reached some hard coded limit and I'm not using Limits: even though setting it to false imposes a limit of 21 quickship:vip default

I have no way of knowing how many shops I have other than the qs.log showing line 192 (last shop I created) - in game players don't have this access.  Instead of informing the player of any maximum limit, all we see is:
![image](https://cloud.githubusercontent.com/assets/10309346/26029085/bf2c95d6-37fa-11e7-880c-232bfebea732.png)
Using SQLite on the shops.db, I have 190 rows of data, though my config.yml shows the below.   Without adding quickshop:vip to my permissions, I can only create 21 shops.  With the permission node and setting to a number more than 190 I've reached some maximum (assuming).  Any idea what might be doing this?

**EDIT:** Not sure what happened but possibly just opening the db with SQLite freed up some process or lock??  I'm able to create more shops including the two in the picture.  BungeeCord Server w/Spigot 1.11.2 latest release on Windows 2010 Professional x64 in case that helps.  Also using Java 1.8.0_131.

**Comment by KaiKikuchi** [comment]
There isn't any "internal limit" other than the integer limit (2,147,483,647). I've seen a lot of servers with over 1000 shops without any issue.

I do not see anything wrong on your screenshot. Maybe you should explain more what exactly you are doing, what you were expecting, and what you exactly got instead? 

Also, try with normal chests.

**Comment by smmmadden** [comment]
You beat me to it. :-)  I figured out that between left clicking the chest and then pressing T for typing in the amount, my mouse pointer moved away from the chest so it lost focus and cancelled the shop creation.  I've been able to continue now that I'm watching the mouse pointer.  Just wasn't noticible when I was doing it.    I think this is safe to close as a user error, though would be nice to tell the user what's wrong (if at all possible, like a lost focus on the object)?
